crates/rpc/beacon/src/handlers/pool.rs

```diff
@@ -98,3 +109,55 @@ pub async fn post_voluntary_exits(
 
     Ok(HttpResponse::Ok())
 }
+
+/// GET /eth/v2/beacon/pool/attester_slashings
+#[get("/beacon/pool/attester_slashings")]
+pub async fn get_pool_attester_slashings(
+    operation_pool: Data<Arc<OperationPool>>,
+) -> Result<impl Responder, ApiError> {
+    let attester_slashings = operation_pool.get_all_attester_slashings();
+    Ok(HttpResponse::Ok().json(DataVersionedResponse::new(attester_slashings)))
+}
+
+/// POST /eth/v2/beacon/pool/attester_slashings
+#[post("/beacon/pool/attester_slashings")]
+pub async fn post_pool_attester_slashings(
+    db: Data<ReamDB>,
+    operation_pool: Data<Arc<OperationPool>>,
+    network_manager: Data<Arc<NetworkManagerService>>,
+    attester_slashing: Json<AttesterSlashing>,
+) -> Result<impl Responder, ApiError> {
+    let attester_slashing = attester_slashing.into_inner();
+
+    let highest_slot = db
+        .slot_index_provider()
+        .get_highest_slot()
+        .map_err(|err| {
+            ApiError::InternalError(format!("Failed to get_highest_slot, error: {err:?}"))
+        })?
+        .ok_or(ApiError::NotFound(
+            "Failed to find highest slot".to_string(),
+        ))?;
+    let beacon_state = get_state_from_id(ID::Slot(highest_slot), &db).await?;
+
+    beacon_state
+        .get_slashable_attester_indices(&attester_slashing)
+        .map_err(|_| {
+            ApiError::BadRequest(
+                "Invalid attester slashing, it will never pass validation so it's rejected"
+                    .to_string(),
+            )
+        })?;
+
+    operation_pool.insert_attester_slashing(attester_slashing.clone());
+
+    network_manager.p2p_sender.send_gossip(GossipMessage {
+        topic: GossipTopic {
+            fork: beacon_state.fork.current_version,
+            kind: GossipTopicKind::AttesterSlashing,
+        },
+        data: attester_slashing.as_ssz_bytes(),
+    });
```

Comment:
```suggestion
    network_manager.p2p_sender.send_gossip(GossipMessage {
        topic: GossipTopic {
            fork: beacon_state.fork.current_version,
            kind: GossipTopicKind::AttesterSlashing,
        },
        data: attester_slashing.as_ssz_bytes(),
    });

    operation_pool.insert_attester_slashing(attester_slashing);
```

this was we can avoid the clone